### ISSUE TITLE
Add strong indication when development build completes

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/metamask-extension#7391]
Ideally a big green log, or maybe even a sound.

Since the snaps-beta is live, we have more devs than ever running development builds of MetaMask, and not knowing when the build is complete is a recurring point of confusion.

**Comment by crazyrabbitLTC** [comment]
Yes, this would be super helpful. :-) I'm still not sure when it's done done. ;-)

**Comment by rekmarks** [comment]
It would also be nice to have a progress bar!

**Comment by dodonpaa** [comment]
when I ran "yarn start", the script was frozen at "scripts:core:dev:standardEntryPoints: Bundle end: "primary". I had to press Ctrl+c to exit the script. I'm not sure if the build was complete. Does anyone know if this is expected behavior or a bug in the build script?
![1CDD63AC-31BB-4E2C-B749-4D16A2FA91F2](https://user-images.githubusercontent.com/64734512/163531943-3f230295-c236-47df-9c23-e979b4b6b708.jpeg)
